cmd/docker/docker.go

```diff
@@ -357,6 +353,39 @@ func runDocker(dockerCli *command.DockerCli) error {
 	return cmd.Execute()
 }
 
+func initializeTracing() error {
```

Comment:
Should we pass the context here that was set on the command?

Reply:
If we have a `context.Context` available, it's probably worth passing through.

That said...I stepped through the source - neither `otlphttp` or `otlpgrpc` actually use it 🙃